### DISCUSSION TITLE
Snapshot Builds

### DISCUSSION THREAD

**Discussion by rwparris2** [mapstruct/mapstruct#3105] [Q&A] [answered]
I have a need to use the new `@TargetPropertyName` annotation. 

So my question has two parts:

1) Are snapshot builds published anywhere? 

2) Is there a roadmap for when that might be released?

**Comment by filiphr** [comment, marked as answer]
>1. Are snapshot builds published anywhere?

Yes SNAPSHOTS are deployed to Sonatype Central Snapshots (https://s01.oss.sonatype.org/content/repositories/snapshots/)


> 2. Is there a roadmap for when that might be released?

Since this project is entirely led in the free time of the members of the team we do not have any special dates for releases. We do want to release an Alpha / Beta of 1.6 soonish, but sometimes we get busy with life and things slow down.

**Reply by rwparris2** [reply]
Thank you filiphr!

I look forward to the next release. Thank you all for your hard work on the project.

**Reply by EvaristeGalois11** [reply]
> Thank you filiphr!
> 
> I look forward to the next release. Thank you all for your hard work on the project.

Have you managed to find the snapshot artifact? I can't find it :/

**Reply by filiphr** [reply]
@EvaristeGalois11 it seems like it is missing in https://s01.oss.sonatype.org/content/repositories/snapshots/, but it is available in https://oss.sonatype.org/content/repositories/snapshots/. I don't know why. I thought that they are the same

**Reply by EvaristeGalois11** [reply]
Thank you @filiphr it works indeed!

Just FYI i found this blogpost that clarifies a bit that they are not in fact the same thing https://central.sonatype.org/news/20210223_new-users-on-s01/#why-are-we-doing-this
